source/_integrations/bme280.markdown

```diff
@@ -187,3 +194,75 @@ It will output a table like this:
 ```
 
 So you can see the sensor address what you are looking for is **0x77** (there are more i2c sensors in that Raspberry Pi).
+
+## SPI Support
+
+### Config example
+
+```yaml
+# Example of customized configuration.yaml entry
+bme280:
+  - name: Stary_Oskol
+    spi_bus: 0
+    spi_dev: 1
+    time_standby: 5
+    oversampling_temperature: 4
+    oversampling_pressure: 4
+    oversampling_humidity: 4
+    monitored_conditions:
+      - temperature
+      - humidity
+      - pressure
+    scan_interval: 40
+```
+
+## Directions for installing SPI support on Raspberry Pi
```

Comment:
We should be clear about which installation type this entails.
For example, these instructions do not apply to users of Home Assistant OS.
How should they handle this?